src/renderer/backend/dx9.rs

```diff
@@ -90,7 +91,17 @@ impl RenderContext for D3D9RenderEngine {
 
 impl RenderEngine for D3D9RenderEngine {
     type RenderTarget = IDirect3DSurface9;
-
+    fn reset(&mut self) {
+        tracing::error!("Calling reset stuff");
+        let mut params = D3DPRESENT_PARAMETERS { ..Default::default() };
+        unsafe {
+            let result = self.device.Reset(&mut params as *mut D3DPRESENT_PARAMETERS);
+            match result {
+                Ok(_) => tracing::info!("All is good"),
+                Err(err) => tracing::error!("Error from reset: {}", err),
+            }
```

Comment:
I'd rather not report the success case at all, and only manage the error case instead. I'd also use the single `e` character name out of style consistency with the codebase.

```suggestion
            if let Err(e) = self.device.Reset(&mut params as *mut D3DPRESENT_PARAMETERS) {
                Err(err) => tracing::error!("Error from reset: {}", err),
            }
```